client/components/data/query-site-stats/index.jsx

```diff
@@ -38,7 +38,7 @@ class QuerySiteStats extends Component {
 
 		this.props.requestSiteStats( siteId, statType, query );
 		this.clearInterval();
-		if ( heartbeat, isAutoRefreshAllowedForQuery( query ) ) {
+		if ( heartbeat && isAutoRefreshAllowedForQuery( query ) ) {
```

Comment:
For reference, [this](https://github.com/Automattic/wp-calypso/pull/11035) is the relevant introduction PR. I couldn't see anything obvious comments regarding the comma.